awesomplete.js

```diff
@@ -40,18 +43,20 @@ var _ = function (input, o) {
 
 	// Create necessary elements
 
-	this.container = $.create("div", {
-		className: "awesomplete",
-		around: input
-	});
+  this.container = $("." + this.containerClass, this.input.parentNode) ||
```

Comment:
Ehm, no not really... 

#16718 is about having a structure where an element must remain a sibling of the input, like in many material-design implementations.

right now, if we have

```html
<div class="some-wrapper-class">
  <input id="search" class="awesomplete someOtherClass" />
  <label for="search">Search for something</label>
</div>
```
Awesomplete will wrap the input with a div, separating it from from the label:

```html
<div class="some-wrapper-class">
  <div class="awesomplete">
    <input id="search" class="awesomplete someOtherClass" />
  </div>
  <label for="search">Search for something</label>
</div>
```

What's needed to resolve #16718 is for `new Awesomeplete('.awsomplete', {containerClass: 'some-wrapper-class'}` to not wrap the input - because it is already wrapped by `some-wrapper-class` - leaving us with the original structure, so that the input and the label remain siblings.

This, inherently, also allows for custom elements, as it just looks at the existing html to see if it can be left as is, or needs modifying, based on a class name. `new Awesomplete` should just leave the following as is, allowing for the wrapper to be an element that isn't a `div`:

```html
<label class="awesomplete">
  <input id="search" class="awesomplete someOtherClass" />
  Search for something
</label>
```